ktor-server/ktor-server-plugins/ktor-server-metrics/jvm/src/io/ktor/server/metrics/dropwizard/DropwizardMetrics.kt

```diff
@@ -79,22 +80,15 @@ public val DropwizardMetrics: ApplicationPlugin<DropwizardMetricsConfig> =
             )
         }
 
-        on(MonitoringEvent(Routing.RoutingCallFinished)) { call ->
-            val routingMetrics = call.attributes.take(routingMetricsKey)
-            val status = call.response.status()?.value ?: 0
-            val statusMeter =
-                pluginConfig.registry.meter(name(pluginConfig.baseName, routingMetrics.name, status.toString()))
-            statusMeter.mark()
-            routingMetrics.context.stop()
-        }
-
         @OptIn(InternalAPI::class)
         on(Metrics) { call ->
             active.inc()
             call.attributes.put(measureKey, CallMeasure(duration.time()))
         }
 
-        on(AfterCall) { call ->
+        on(ResponseSent) { call ->
+            callFinished(call)
```

Comment:
can you please do a small cleanup here and extract all `pluginConfig` properties into local vals? config is a mutable object, and it's better to capture all its properties during the plugin installation.